lib/notification_service/notification_service/cli/commands/server_command.py

```diff
@@ -90,22 +97,25 @@ def server_stop(args):
 
     try:
         os.kill(pid, signal.SIGTERM)
-        logger.info("Notification server pid: {} stopped".format(pid))
     except Exception:
         logger.error("Failed to stop Notification server (pid: {}) with SIGTERM. Try to send SIGKILL".format(pid))
```

Comment:
This log could be warn ,because when stopping Notification server with SIGKILL signal successly, this log is confusing for users.

Reply:
fixed in e0be4eb